pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -57,6 +66,31 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
 
 	jobframework.ApplyDefaultForSuspend(jobSet, w.manageJobsWithoutQueueName)
+	if features.Enabled(features.MultiKueue) {
+		localQueueName, found := jobSet.Labels[constants.QueueLabel]
+		if !found {
+			return nil
+		}
+		clusterQueueName, err := w.queues.ClusterQueueFromLocalQueue(fmt.Sprintf("%s/%s", jobSet.ObjectMeta.Namespace, localQueueName))
```

Comment:
I don't think so. I would vote for changing `ClusterQueueFromLocalQueue` to accept `namespace` and `name`. This would also simplify [here](https://github.com/kubernetes-sigs/kueue/blob/0446cb5e54655d3057e2e96a50de1f9da03b0a0e/pkg/visibility/api/rest/pending_workloads_lq.go#L70).